models/user_client.py

```diff
@@ -159,7 +159,7 @@ def _group_code(self):
     nacionalidad = fields.Char("Nacionalidad")
 
     # id externo en tabla de res_partner
-    client_export_id = fields.Char("Export ID")
+    client_export_id = fields.Char("Export ID", default="None")
```

Comment:
Ese odoo hasta el None tiene que esta en string wooo que raro!

Reply:
Eso ya es hacky, otro modelo depende de esa propiedad y truena si esta nulo, es un fallback para las entidades creadas en Odoo y no importadas.

Reply:
Okay tomare en cuenta para revisar las practicas de odoo respecto a modelos
"When to Use Null and Blank by Field"
En django tenemos null = True, Blank= True, default
Null si puede existir si el campo es objeto, como fecha, o relacional.
Blank es mas front, significa que el campo no es requerido.
y Default es como odoo para definir según el tipo de field
Y usamos una tablita para esas convenciones te la pongo en este link por si sirve en futuras referencias https://imgur.com/a/R2WGU
Claro, no son iguales pero debemos adoptar algo similar, por ejemplo porque no poner cadena vacia de default `default=""`

Reply:
Gracias por la referencia, no use blank porque tenia pendiente que lo interpretara como vacio. Vale la pena probarlo como dices.